src/pointerAction.js

```diff
@@ -98,7 +98,11 @@ function watchDraggedElement() {
         MIN_OBSERVATION_INTERVAL_MS,
         ...Array.from(dropZones.keys()).map(dz => dzToConfig.get(dz).dropAnimationDurationMs)
     );
-    observe(draggedEl, dropZones, observationIntervalMs * 1.07);
+    const useTouchPosition = Array.from(dropZones.keys())
```

Comment:
looks like using this option for one dndzone will apply to all. this could be kind of problematic because maybe for some it would not be ideal